modules/lvms-creating-volume-clones.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-creating-volume-clones_{context}"]
+= Creating volume clones
+
+To create a clone of a persistent volume claim (PVC), you must create a `PersistentVolumeClaim` custom resource (CR) in the namespace where you created the source PVC.
+
+[IMPORTANT]
+====
+The cloned PVC has write access.
+====
+
+.Prerequisites
+
+* You ensured that the source PVC is in `Bound` state. This is required for a consistent clone.
+
+.Procedure
+
+. Log in to the OpenShift CLI (`oc`).
+
+. Create a `PersistentVolumeClaim` custom resource (CR) YAML file.
```

Comment:
Hi @suleymanakbas91 
Since we need to differentiate between a PVC resource and the YAML configuration to create a PVC resource, can we use the term `object` instead of custom resource to refer to such YAML configurations for PVC?

For example:

Create a `PersistentVolumeClaim` object.

Reply:
Yes, that's also fine :)

Reply:
Updated all the instances of PVCs, volume snapshots, and volume clones referred as CR to `object`.